backends/arm/test/ops/test_rsqrt.py

```diff
@@ -104,3 +110,96 @@ def test_rsqrt_vgf_INT(test_tensor: torch.Tensor):
         tosa_version="TOSA-1.0+INT",
     )
     pipeline.run()
+
+
+def get_symmetric_a16w8_rsqrt_quantizer(
+    u55_config=False, per_channel_quantization=False
+):
+    tosa_version = conftest.get_option("tosa_version")
+    tosa_profiles = {
+        "1.0": TosaSpecification.create_from_string("TOSA-1.0+INT+int16"),
+    }
+
+    quantizer = TOSAQuantizer(tosa_profiles[tosa_version])
+    quantizer.set_global(
+        get_symmetric_a16w8_quantization_config(is_per_channel=per_channel_quantization)
+    )
+
+    return Quantize(
+        quantizer,
+        get_symmetric_a16w8_quantization_config(
+            is_per_channel=per_channel_quantization
+        ),
+    )
+
+
+@common.parametrize("test_tensor", Rsqrt.test_parameters)
+def test_rsqrt_int16_tosa_INT(test_tensor: torch.Tensor):
```

Comment:
It looks like there are some failures on this particular testcase. For rand_4d_1 and rand _4d_2. It might be best to see these as expected failures like below. Thanks for adding this!